multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/security/UriAuthorizationFilter.java

```diff
@@ -23,9 +23,9 @@ public interface UriAuthorizationFilter {
     default void auditLogLoginAttempt(String username, String spaceGuid, String message) {
         String performedAction = MessageFormat.format(message, username, spaceGuid);
         AuditLoggingProvider.getFacade()
-                .logSecurityIncident(new ExtentensionAuditLog(SecurityContextUtil.getUsername(),
-                        spaceGuid,
-                        performedAction,
-                        Messages.LOGIN_ATTEMPT_AUDIT_LOG_CONFIG));
+                            .logSecurityIncident(new AuditLogConfiguration(SecurityContextUtil.getUsername(),
```

Comment:
My idea is to fully remove this static `AuditLoggingProvider` and use only beans injection.